src/entities/Game.entity.ts

```diff
@@ -141,7 +141,7 @@ export class Game extends BaseEntity {
   @ApiProperty({ description: 'Cover URL' })
   @Column()
   @IsUrl()
-  @IsNotEmpty()
+  @IsOptional()
```

Comment:
Cover URL will be generated if kind of project is `Minetest`.How about `RPGMaker2k3`?